conda_smithy/templates/azure-pipelines-win.yml.tmpl

```diff
@@ -54,12 +54,13 @@ jobs:
 {%- else %}
 {%- set BOA="" %}
 {%- endif %}
-    - task: CondaEnvironment@1
-      inputs:
-        packageSpecs: 'python=3.9 conda-build conda pip {{ BOA }}{{ " ".join(remote_ci_setup) }}' # Optional
-        installOptions: "-c conda-forge"
-        updateConda: true
-      displayName: Install conda-build and activate environment
+    - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+      displayName: Add conda to PATH
+
+    - script: |
+        call activate base
+        conda.exe install 'python=3.9' conda-build conda pip {{ BOA }}'{{ " ".join(remote_ci_setup) }}' -c conda-forge --strict-channel-priority
```

Comment:
@wolfv, I've moved to calling conda directly as the azure task was problematic. What do you think?

Reply:
Looks great!

Reply:
Can you add a news entry? Then we can merge.

Reply:
I added a news entry but I think your other fix looks even better. Thanks!